### PR TITLE
firewalld: Use nft instead of iptables for 15.3

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/80966
Verification:
SLE: http://kazhua.qa.suse.de/tests/320
openSUSE JeOS: http://kazhua.qa.suse.de/tests/321

### CODE REVIEW

File: tests/console/firewalld.pm [hunk 1]
```diff
@@ -18,6 +18,10 @@ use testapi;
 use utils qw(systemctl zypper_call);
 use version_utils qw(is_sle is_leap is_tumbleweed);
 
+sub has_iptables {
+    return (!is_tumbleweed && !is_sle('15-SP3+') && !is_leap('15.3+'));
```
Comment (Vogtinator) [inline tests/console/firewalld.pm:22]: Simplify and avoid `is_tumbleweed`:

```suggestion
    return is_sle('<15-SP3') || is_leap('<15.3');
```

File: tests/console/firewalld.pm [hunk 2]
```diff
@@ -18,6 +18,10 @@ use testapi;
 use utils qw(systemctl zypper_call);
 use version_utils qw(is_sle is_leap is_tumbleweed);
 
+sub has_iptables {
```
Comment (Vogtinator) [inline tests/console/firewalld.pm:21]: Arguably they still "have" iptables, so maybe:

```suggestion
sub uses_iptables {
```

**Review by Vogtinator — COMMENTED** [review]
LGTM otherwise

File: tests/console/firewalld.pm (hunk 1 above)
Comment (foursixnine) [reply tests/console/firewalld.pm:22]: See #12345

**Comment by asdil12** [comment]
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12345 will contain this patch.

**Pull request closed without merging**